src/DTO/LoginDTO.ts

```diff
@@ -0,0 +1,8 @@
+import {IsNotEmpty} from "class-validator";
```

Comment:
Have created a DTO (data transfer object) for the login - this means that we can validate the input. And have a nice place to put our email and password.